src/views/HowToPlay/Layout/StyledInstructionsLayout.tsx

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+
+const StyledInstructionsLayout = styled.div`
+  .container {
+    display: flex;
+    align-items: center;
+    margin: 25px 0 0 3em;
```

Comment:
Let's get rid of this margin property. The page looks better with more concise spacing between the `.container` elements. And if you want to set left/right margin, you should do it on the outer div like you did on line 19.

```suggestion
```